src/main/java/com/infobip/bitbucket/ProjectKeyValidator.java

```diff
@@ -13,26 +13,25 @@
  # See the License for the specific language governing permissions and
  # limitations under the License.
  */
-package com.infobip.stash;
+package com.infobip.bitbucket;
 
-import javax.annotation.Nonnull;
-import com.atlassian.stash.repository.Repository;
-import com.atlassian.stash.setting.RepositorySettingsValidator;
-import com.atlassian.stash.setting.Settings;
-import com.atlassian.stash.setting.SettingsValidationErrors;
+import com.atlassian.bitbucket.repository.Repository;
+import com.atlassian.bitbucket.setting.RepositorySettingsValidator;
+import com.atlassian.bitbucket.setting.Settings;
+import com.atlassian.bitbucket.setting.SettingsValidationErrors;
 import com.infobip.jira.ProjectKey;
 
 class ProjectKeyValidator implements RepositorySettingsValidator {
 
 	static final String SETTINGS_KEY = "jira-project-key";
 
 	@Override
-	public void validate(@Nonnull Settings settings,
```

Comment:
it was never validated actually (it's not bean validation annotation), only used by IDEA and findbugs to issue a warning if caller passes a null. It still works as the inherited method from RepositorySettingsValidator contains Nonnull. 
IDEA does issue a warning on the parameter:
"Not annotated parameter overrides @Nonnull parameter".

I have now added package-info which contains ParametersAreNonnullByDefault so it doesn't emit that warning anymore.